docs/topics/gradle.md

```diff
@@ -316,10 +293,51 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
 >
 {type="note"}
 
+Additionally, you can use the `kotlin-test` dependency in any shared or platform-specific source set.
+
+For Kotlin/JVM Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to JUnit 4.
```

Comment:
_kotlin("test")_ (double quotes) is better because it works in both Groovy and Kotlin scripts.

Reply:
> the `kotlin('test')` dependency resolves to JUnit 4.

This new wording looks confusing to me. `kotlin("test")` doesn't resolve to JUnit 4, it resolves to the artifact of `kotlin-test` for JUnit 4. We called it "variant" before as it is called in the standard Gradle terminology of variant-aware dependency resolution.

Reply:
Thank you! I'm going to return back "the `kotlin("test")` dependency resolves to the variant for 
JUnit 4, namely `kotlin-test-junit`"